Add a `usage` section to RPM/DEB doc (#204)
tarantool/cartridge-cli: README.md

export PATH=$PWD/.rocks/bin/:$PATH
```

Now you can create and start your first application!
Go to the [next](#quick-start) section and try it.

## Quick start

To create your first application:

```sh
cartridge create --name myapp
```

Let's go inside:

```sh
cd myapp
```

Now build the application and start it:

```sh
cartridge build
cartridge start
```

> That's all! You can visit http://localhost:8081 and see your application Admin Web UI:

Reviewer: ```suggestion
That's all! You can visit http://localhost:8081 and see your application's cluster management web interface:
```

* `--use-docker` (ignored for `docker`) forces to build the application in Docker.

* `--tag` (used for `docker`) is the tag of the Docker image that results from
  `pack docker`.

> * `--build-from` (common for all distribution types, used on build in Docker) is

Reviewer: ```suggestion
* `--build-from` (common for all distribution types, used for building in Docker) is
```

  the path to the base Dockerfile of the build image.
  Defaults to `Dockerfile.build.cartridge` in the application root.

> * `--sdk-local` (common for all distribution types, used on build in Docker) is a

Reviewer: ```suggestion
* `--sdk-local` (common for all distribution types, used for building in Docker) is a
```

  flag that indicates if the SDK from the local machine should be delivered in the
  result artifact.

> * `--sdk-path` (common for all distribution types, used on build in Docker)is the

Reviewer: ```suggestion
* `--sdk-path` (common for all distribution types, used for building in Docker) is the
```

The result artifact name is `<name>-<version>[-<suffix>].{rpm,deb}`.

#### Usage example

After package installation you need to specify configuration for instances to start.

For example, if your application is named `myapp` and you want to start two instances,

> place a `myapp.yml` file in `/etc/tarantool/conf.d` directory.

Reviewer: ``
```suggestion
put the `myapp.yml` file into the `/etc/tarantool/conf.d` directory.
```

```yaml
myapp:
  cluster_cookie: secret-cookie

myapp.instance-1:
  http_port: 8081
  advertise_uri: localhost:3301

myapp.instance-2:
  http_port: 8082
  advertise_uri: localhost:3302
```

> More details about instances configuration you can find in the

Reviewer: ```suggestion
For more details about instances configuration, see the
```

> [documentation](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_dev/#configuring-instances).

Reviewer: ```suggestion
[documentation](https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_dev/#configuring-instances).
```

> Now, start configured instances:

Reviewer: ```suggestion
Now, start the configured instances:
```
